package.json

```diff
@@ -41,15 +44,25 @@
     }
   ],
   "devDependencies": {
+    "@babel/preset-typescript": "^7.18.6",
     "@size-limit/preset-small-lib": "^8.1.0",
     "@tsconfig/recommended": "^1.0.1",
-    "dts-cli": "^1.6.0",
+    "@types/node": "^18.13.0",
+    "@typescript-eslint/eslint-plugin": "^5.51.0",
+    "@typescript-eslint/parser": "^5.51.0",
+    "eslint": "^8.33.0",
     "husky": "^8.0.2",
     "size-limit": "^8.1.0",
-    "typescript": "^4.9.3"
+    "ts-jest": "^29.0.5",
+    "typescript": "^4.9.5"
   },
   "dependencies": {
+    "@babel/preset-env": "^7.20.2",
     "@mento-protocol/mento-core-ts": "^1.0.0",
-    "ethers": "^5.7.2"
+    "@types/jest": "^29.4.0",
```

Comment:
dev dependency?

Reply:
Yeah, everything except @mento-protocol/mento-core-ts and ethers should be dev dependencies.
In fact, I'd recommend making `ethers` dev dep + peer dep instead of a direct dep, because pretty much every DApp using this SDK will already have a version of ethers. Adding it here too risks having two-versions get bundled into the DApp. @nvtaveras WDYT?

Reply:
Sounds reasonable to me! However I don't live in NPM land a lot, are there any common practices with peer dependencies like to mention them in the readme or are people expected to look at the package.json / warnings when installing the package and tell for themselves?